applications/nrf5340_audio/src/audio/streamctrl_broadcast_sink.c

```diff
@@ -628,16 +566,23 @@ int streamctrl_start(void)
 	ret = zbus_subscribers_create();
 	ERR_CHK_MSG(ret, "Failed to create zbus subscriber threads");
 
-	ret = bt_rend_init();
-	ERR_CHK(ret);
+	ret = zbus_link();
```

Comment:
Nitpick: Would we use zbus_setup() or zbus_init() instead of link?

Reply:
Changed name to "zbus_link_producers_observers"